### ISSUE TITLE
Master: PRECSALT Sequentional RESTART Arrays

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-simulators#3769]
For the SPE1CASE1_PRECSALT.data test case the RESTART file contains an ICON with infinite values and other values. Also what should be the data in this array. 

![image](https://user-images.githubusercontent.com/17958927/149089089-eedcf6a2-24fe-4fe5-a287-d0b3d2202c06.png)

Secondly, there is an XWEL array that seems to have garbage as well. What should be the name of this array?

![image](https://user-images.githubusercontent.com/17958927/149089153-d561158e-ac7c-45fb-9b2d-832f9ba03bff.png)

**Comment by bska** [comment]
> the RESTART file contains an ICON with infinite values and other values.

Note that `ICON` contains integer data per well/reservoir connection.  We (currently) output 25 integers per (potential) reservoir connection, of which there are exactly [two](https://github.com/OPM/opm-tests/blob/a0198b2ac1c06f1448c8213976563114590017d1/spe1_precsalt/SPE1CASE1_PRECSALT.DATA#L54-L63) in `SPE1CASE1_PRECSALT` (one for each of the two wells).  In other words, `ICON` contains a total of 50 integer items, most of which happen to be zero in current Flow.  It is not meaningful to try to plot `ICON` as a cell result.

> there is an XWEL array that seems to have garbage as well.

`XWEL` contains 58 double precision dynamic solution data items for each (potential) well in the model.  One of those items will be the total gas production/injection volume which is typically large.  In the case of `SPE1CASE1_PRECSALT` the restart file will have an `XWEL` array of size 116 for each restart step.  It too is not a cell result and it does not make sense to try to plot it as such.

**Comment by OPMUSER** [comment]
Yes, you would think having written this I would remember what those arrays where :-) - I guess I was distracted by the parallel runs not working.

Have we tried using HIDDEN arrays to hide this data? Assuming of course OPM ResInsight honors theys keywords in the RESTART file. 

![image](https://user-images.githubusercontent.com/17958927/149132298-96b6e395-534f-4c76-a3f0-64a25fca9832.png)
